packages/dashboard-v2/package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "scripts": {
     "develop": "gatsby develop",
-    "develop:secure": "gatsby develop --https --host=local.skynetpro.net -p=443",
+    "develop:secure": "dotenv -e .env.development -- gatsby develop --https -p=443",
```

Comment:
you shouldn't need to use dotenv explicitly here, just use `NODE_ENV=development`

Reply:
Gatsby reads `GATSBY_HOST` too late, at least in `gatsby develop`. Starts on `localhost` instead of defined hostname.